modules/sdk-coin-avaxp/src/transaction.ts

```diff
@@ -48,20 +75,20 @@ export class Transaction extends BaseTransaction {
    * @param {KeyPair} keyPair
    */
   sign(keyPair: KeyPair): void {
-    const keys = keyPair.getKeys();
-    if (!keys.prv) {
+    // getPrivateKey returns type 32 byte, not PrivateKey- which is the right format
```

Comment:
Who is this question addressed to?

Reply:
removed, was a comment to myself that i forgot to remove